### ISSUE TITLE
can't load automerge repo

### ISSUE THREAD

**Issue by mhuangwm** [automerge/automerge-repo#171]
Hi, just playing around with automerge a little bit. Trying to follow the docs in a simple nodejs app (i.e no react or webserver ect..). But I can't seem to load an automerge repo in subsequent runs. Any insight would be appreciated. Thankyou

```
import { isValidAutomergeUrl, Repo } from '@automerge/automerge-repo'
import { BroadcastChannelNetworkAdapter } from '@automerge/automerge-repo-network-broadcastchannel'
import { NodeFSStorageAdapter } from "@automerge/automerge-repo-storage-nodefs"
import {next as A} from "@automerge/automerge" //why `next`? See the the "next" section of the conceptual overview

const repo = new Repo({
    network: [new BroadcastChannelNetworkAdapter()],
    storage: new NodeFSStorageAdapter()
})

const url = "automerge:2EZTTsmudNkQF7esKjhzuUhRivmD"
let handle
if (isValidAutomergeUrl(url)) {
    handle = repo.find(url)
} else {
    handle = repo.create()
    console.log(handle.url)
    handle.change(d => d.counter = new A.Counter())
}

function handleState(){
    console.log(handle.url, handle.state)
    setTimeout(function () {
        handleState()
    }, 1000)
}

handleState()
```

```
automerge:2EZTTsmudNkQF7esKjhzuUhRivmD loading
automerge:2EZTTsmudNkQF7esKjhzuUhRivmD unavailable
automerge:2EZTTsmudNkQF7esKjhzuUhRivmD unavailable
```

**Comment by pvh** [comment]
Interesting! It's possible that the document wasn't saved at all and so there's no document to load -- you see "loading" because that's where the state machine starts but after checking the network and disk nothing is found so it transitions to "unavailable."

Can you confirm you have the data somewhere it should be able to see?

**Comment by mhuangwm** [comment]
```
➜  ZTTsmudNkQF7esKjhzuUhRivmD git:(main) ✗ pwd
<>/automerge-repo-data/2E/ZTTsmudNkQF7esKjhzuUhRivmD
➜  ZTTsmudNkQF7esKjhzuUhRivmD git:(main) ✗ tree
.
├── incremental
│   └── 2b4ef81c9d7bd060f97f1e8b0678fbc487e31149242cb693634132726c90adf6
└── snapshot
    └── 0eece75a15db440ce375d0ae6b8718d7cdac5baa6592be4c9426941a3e3329bc

3 directories, 2 files
```

The two files contain binary data

**Comment by pvh** [comment]
Great, I confirmed that I'm seeing the same bug here. I'll see if I can figure out what's up.

**Comment by pvh** [comment]
Thanks for the report @mhuangwm; this is fixed with #174 and I'll push a point release as soon as someone else has doublechecked my work. (I'd rather not make things worse by accident.)
